static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -232,7 +236,8 @@ define([
             window.googletag.cmd.push(defineSlots);
 
             // We want to run lazy load if user is in the main test or if there is a switch on
-            if (ab.shouldRunTest('Viewability', 'variant') || isLzAdsSwitchOn()) {
+            // We do not want lazy loading on pageskins because it messes up the roadblock
+            if ((ab.shouldRunTest('Viewability', 'variant') || isLzAdsSwitchOn()) && !isPageSkinned()) {
```

Comment:
I know that you used similar patter to `isLzAdsSwitchOn` but they are both returning only variable so wrapping them into functions is unnecessary. I would remove both functions and replace it just with config variables.